tutorials/custom-migration-tutorial.md

```diff
@@ -0,0 +1,122 @@
+# Upgrading to v1.1.1 using a Custom NamedKey Convention
+
+This tutorial uses the Casper command-line client to upgrade *and* migrate from an NFT contract installed using release [v1.0.0](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/tag/v1.0.0) and customized NamedKeys. The outcome of the upgrade will be a [v1.1.1](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/tag/v1.1.1) NFT contract with its storage structure migrated to the new format.
+
+## Prerequisites
+
+- You have previously installed a CEP-78 contract using release v1.0.0 on a Casper network. Thus, you have experience with the [Casper CEP-78 NFT Standard](https://github.com/casper-ecosystem/cep-78-enhanced-nft/), the Casper command-line client, and interacting with a Casper network.
+- Your v1.0.0 NFT contract instance uses custom NamedKeys for the contract package hash and contract package access URef. An example of how the NamedKeys could have been customized is found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/dev/test-contracts/mangle_named_keys/src/main.rs).
+- You have the v1.0.0 contract package hash stored under a custom NamedKey in the account that installed the contract.
+- You have the v1.0.0 contract package access URef stored under a custom NamedKey in the account that installed the contract.
+- You understand what is [New in Version 1.1](https://github.com/casper-ecosystem/cep-78-enhanced-nft/#new-in-version-11) of the CEP-78 Enhanced NFT Standard.
+
+## Upgrading and Migrating Terminology
+
+The upgrade to version 1.1.1 involves a data migration to a new [page system](https://github.com/casper-ecosystem/cep-78-enhanced-nft#the-cep-78-page-system) tracking token ownership. The usual [upgrade](https://docs.casperlabs.io/dapp-dev-guide/writing-contracts/upgrading-contracts/) process triggers the data migration. For more information, see [Standard Migration Tutorial](standard-migration-tutorial.md#upgrading-and-migrating-terminology).
+
+## Steps to Upgrade to Version 1.1.1
+
+Navigate to the [v1.1.1 release](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/tag/v1.1.1) and download the [cep-78-wasm.tar.gz](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/download/v1.1.1/cep-78-wasm.tar.gz) file. Unarchive it to have access to the Wasm files provided. The Rust source code is also available if you would like to build and test the contract according to the Makefile provided. 
+
+The `cep-78-wasm` folder contains the `contract.wasm` to send to the network to upgrade the NFT contract to version 1.1.1. In addition, you will see other useful Wasm files to interact with the contract once it is stored on-chain.
```

Comment:
Double check the folder name

Reply:
Done. It is correct once unzipped. 
<img width="277" alt="Screen Shot 2023-01-17 at 9 16 42 PM" src="https://user-images.githubusercontent.com/4185994/213002717-f6edd5b9-882d-4a27-b518-7d4a8f6932fe.png">